packages/core/src/loadConfig.ts

```diff
@@ -149,10 +158,16 @@ export async function loadConfig({
           `Failed to load file with native loader: ${color.dim(configFilePath)}`,
         );
         throw err;
+      } else if (loader === 'auto') {
+        logger.debug(
+          `failed to load file with native loader, fallback to jiti: ${color.dim(configFilePath)}`,
+        );
+        logger.debug(err);
+      } else {
+        logger.debug(
+          `failed to load file with dynamic import: ${color.dim(configFilePath)}`,
+        );
```

Comment:
Can we merge the two error logs? For example:

```suggestion
      } else {
        logger.debug(
          `failed to load file with native dynamic import: ${color.dim(configFilePath)}`,
        logger.debug(err);
        );
```

Reply:
Vibe coding is indeed quite verbose.